cpustat.cpp

```diff
@@ -82,10 +82,10 @@ void CpuStatPrivate::updateSources()
         int dash = range.indexOf(QLatin1Char('-'));
         if (dash != -1)
         {
-            uint min = range.leftRef(dash).toUInt(&ok);
+            uint min = range.left(dash).toUInt(&ok);
```

Comment:
Since the index is inside the bounds, we can use `QString::first`; Qt doc recommends it (→ https://doc.qt.io/qt-6/qstring.html#left):

```c++
            uint min = range.first(dash).toUInt(&ok);
```